conda/environments/all_cuda-125_arch-x86_64.yaml

```diff
@@ -27,8 +27,6 @@ dependencies:
 - ipython
 - ipywidgets
 - libcudf==24.10.*,>=0.0.0a0
-- libcuspatial-tests==24.10.*,>=0.0.0a0
-- libcuspatial==24.10.*,>=0.0.0a0
```

Comment:
See the build instructions in the development docs: https://github.com/rapidsai/cuspatial/blob/1544e7bd16f11457436d1fb6a43da0db3aa9af57/docs/source/developer_guide/build.md#L15-L32

1. they recommend starting by creating a conda environment from one of the files in `conda/environments`
2. they recommend building `libcuspatial` and its tests from source

Those steps are currently broken.

The `libcuspatial-tests` conda package is not distributed anywhere, so that env creation will fail like this:

```text
Solving environment: failed

ResolvePackageNotFound: 
  - libcuspatial-tests[version='24.10.*,>=0.0.0a0']
```

Here, I'm recommending dropping those packages from these files. Similar to how for example you won't find e.g. `libcuml` in these files in the `cuml` repo ([code link](https://github.com/rapidsai/cuml/blob/2f1157047c4db32897bcfe2c477c55c0708a34ec/conda/environments/all_cuda-118_arch-x86_64.yaml)) or `libcuml` in the `cudf` repo ([code link](https://github.com/rapidsai/cudf/blob/2d6758f39592e6296a042eb8e771171c50899013/conda/environments/all_cuda-125_arch-x86_64.yaml)).